### PR TITLE
#984 Updating `indexedAt` timestamps

### LINKED ISSUE #984: Updating post timestamps requires refresh from server
**Describe the bug**

Post times shown in the timeline do not update on the client without a refresh.

When the page is pulled up from the background the client should compare local time to the post timestamp and do a local computation to correctly set the post time.

**To Reproduce**

1. Load a post from the "Newskies" feed -- leave it alone for a few minutes... note that the timestamps displayed in the UI do not change over time.

1. Load a post from the "Newskies" feed -- switch tabs. Come back a few minutes later.... note that the timestamps have not been updated.

**Expected behavior**

When the page is pulled up from the background the client should compare local time to the post timestamp and do a local computation to correctly set the post time.

**Screenshots**

image

The post at the top of the feed was not posted 50s ago, it was posted a few minutes ago. Despite that, the time since post display reads '50s'.

**Details**

 - Platform: web, android
 - Platform version:  ???
 - App version: ???

**Additional context**

### PR DESCRIPTION
**Pull request by estrattonbailey**
Fixes #984

This PR provides a little util component that updates a "time ago" value based on an initial starting timestamp.

I also considered using a single global `setInterval` in a mobx store — or using an event emitter — and subscribe to tick events, but decided this was simpler. I'm not sure of the perf difference between 100 `setIntervals` and 100 event subscribers, but I'd guess it's not a significant difference. Lmk if y'all disagree ✌️

### CODE REVIEW

**Review by ansh — COMMENTED** [review]
Works as expected. Will leave it to @pfrazee to decide if having these many `setIntervals` is good or not. Not sure about the performance considerations either. I think it's only a problem on Android due to bad CPUs.

File: src/state/models/ui/shell.ts [hunk 1]
```diff
@@ -224,13 +224,16 @@ export class ShellUiModel {
   activeLightbox: ProfileImageLightbox | ImagesLightbox | null = null
   isComposerActive = false
   composerOpts: ComposerOpts | undefined
+  tickEveryMinute = Date.now()
```
Comment (estrattonbailey) [inline src/state/models/ui/shell.ts:227]: I feel like @pfrazee had a better name when he mentioned this to me but it left my brain immediately :P 

Ideas? Figured maybe we'd have more than one interval in the future for seconds/hours, but maybe not necessary to be so specific in naming.

Comment (pfrazee) [reply src/state/models/ui/shell.ts:227]: Eh this works for now

**Review by pfrazee — APPROVED** [review]
Looks good!

**Pull request merged**